### ISSUE TITLE
Backend link not working

### ISSUE THREAD

**Issue by justnajm** [AsgardCms/Platform#703]
### Summary of issue
I have installed fresh copy and my url is: 
http://localhost/asgardcms/public/
the url is working fine but backend url is not working none of them:
http://localhost/asgardcms/public/backend/
http://localhost/asgardcms/public/en/backend/

### Steps to reproduce
1. Install the copy using composer method.
2. artisan install
3. visit the url

### System Details

- Windows
- PHP Version: 7.1
- Laravel Version: 5.5 (which comes with asgardcms)
- `Asgardcms/Platform` version : 3.0

**Comment by nWidart** [comment]
hi, you shouldn't have public in your urls. having 'backend' in there will probably cause issues too.

**Comment by justnajm** [comment]
Thanks friend,

yes I tried that before, but I figure out that issue is laravel itself, laravel do not have index.php skip in htaccess, because of this I always have to use index.php before any route.
It worked like this: http://localhost/asgardcms/public/index.php/backend/

Now I am having error with theme "Undefined variable: alternate (View: web\asgardcms\Themes\Flatly\views\layouts\master.blade.php)"

**Comment by arturmamedov** [comment]
> alternate

Yes there is a bug also for me, remove the section with alternate loop in master layout
